include/tvm/tir/transform.h

```diff
@@ -709,6 +709,12 @@ TVM_DLL Pass ManifestSharedMemoryLocalStage();
  */
 TVM_DLL Pass InstrumentProfileIntrinsics();
 
+/*!
+ * \brief Create default schedule for PrimFuncs to run on cuda device.
+ * \return The Pass.
+ */
+TVM_DLL Pass DefaultSchedule();
```

Comment:
Should we call it `DefaultCudaSchedule` or `DefaultGPUSchedule` to indicate its specifically for thread bindings?

Reply:
I'm open to naming options, `DefaultCudaSchedule` sounds like a better version given it's only targeting cuda for now.

Reply:
I do think that would be a better name, `DefaultSchedule~ makes it sound like it would be useful for cpu too.

Reply:
Though this pass is intended only for CUDA backend at this moment, I suppose it will actually work for some other GPU backends, as the tasks tuned by MetaSchedule on these backends are all following the same schedule rule.
https://github.com/apache/tvm/blob/main/src/meta_schedule/utils.h#L512

So I personally would prefer the name of “DefaultGPUSchedule” and list the GPU backends this pass supports in the doc string. In the pass we have already checked if the target contains attribute “maximum # of threads per block.” We can fall back with no change if the target doesn’t have this attribute.

Reply:
I agree DefaultGPUSchedule seems to be a good name here

Reply:
It appears that there is consensus among the contributors that the current name DefaultSchedule is not clear enough, and many prefer the new name DefaultGPUSchedule as it allows for potential expansion to other GPU targets in the future. As a result, I have renamed it to DefaultGPUSchedule for the time being, but I am still open to other suggestions for naming.